app/controllers/projects_controller.rb

```diff
@@ -18,7 +18,10 @@ def get_projects
   # GET /projects/1
   # GET /projects/1.json
   def show
-    @project.increase_views(current_user)
+    if current_visit && Ahoy::Event.exists?(visit_id: current_visit.id)
```

Comment:
@animysore Correct me if I'm wrong, but should this check be `current_visit && !(Ahoy::Event.exists?(visit_id: current_visit.id))`. 

Moreover, it should be `current_visit && !(Ahoy::Event.exists?(visit_id: current_visit.id, name: "Visited project #{@project.id}"))` because if a user visits one project URL and after that open another different one then that is counted as the same visit for ahoy. I verified this locally. Could you also test this once?

Reply:
@ankitkataria I see you made a PR with this. Shall I close this ?

Reply:
Let's keep this for reference until the other PR gets merged. We can close it after that. :slightly_smiling_face: